packages/example-next-13-with-pages/src/i18n.tsx

```diff
@@ -0,0 +1,11 @@
+import {NextIntlConfig} from 'next-intl';
+
+const i18n: NextIntlConfig = {
+  locales: ['en', 'de'],
+  defaultLocale: 'en',
+  async getMessages({locale}) {
+    return (await import(`../messages/${locale}.json`)).default;
+  }
+};
+
+export default i18n;
```

Comment:
This config has changed a bit. If you rebase your branch and check out the latest docs, you can use the new format.